PowerUp/app/src/main/java/powerup/systers/com/IntroActivity.java

```diff
@@ -0,0 +1,63 @@
+package powerup.systers.com;
+
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.graphics.Color;
+import android.os.Bundle;
+
+import com.github.paolorotolo.appintro.AppIntro;
+import com.github.paolorotolo.appintro.AppIntroFragment;
+
+public class IntroActivity extends AppIntro {
+
+    private SharedPreferences settings;
+    private boolean firstRun;
```

Comment:
Change this variable name to `isFirstRun` so it's obvious it's a boolean from its name